balloon/history/tree.go

```diff
@@ -53,7 +61,7 @@ func NewHistoryTree(hasherF func() hashing.Hasher, store storage.Store, cacheSiz
 // with the storage mutations to be done at balloon level.
 func (t *HistoryTree) Add(eventDigest hashing.Digest, version uint64) (hashing.Digest, []*storage.Mutation, error) {
 
-	// log.Debugf("Adding new event digest %x with version %d", eventDigest, version)
+	// t.log.Tracef("Adding new event digest %x with version %d", eventDigest, version)
```

Comment:
Do we need a log level which we expect to be commented out? :P

Reply:
It depends on the loss of performance we are determined to accept.